webapp/src/index.js

```diff
@@ -38,6 +39,9 @@ class PluginClass {
         registry.registerBottomTeamSidebarComponent(TeamSidebar);
         registry.registerPopoverUserAttributesComponent(UserAttribute);
 
+        const hooks = new Hooks(store);
+        registry.registerSlashCommandWillBePostedHook(hooks.slashCommandWillBePostedHook);
```

Comment:
There are other places to handle for the connect case. Here's one:

https://github.com/mattermost/mattermost-plugin-gitlab/blob/7190bfda896caae089c36c0e7a65ca907ef1403c/webapp/src/components/sidebar_buttons/sidebar_buttons.jsx#L73

There may just be two, but I would look through the webapp directory to verify if there are more

Reply:
@mickmister We can connect our account using the RHS as well. We handled both cases now.